app/root.tsx

```diff
@@ -253,10 +253,11 @@ export function Layout({ children }: { children?: ReactNode }) {
         <DevBanner />
         <Header />
         <NewsBanner>
-          New! October 18 GCN Classic Outage and Schema v4.2.0. See{' '}
+          VTP Migration Survey and Retirement of Legacy Circular Email
+          Submission Address. See{' '}
```

Comment:
Our users may not know what VTP means, even if they are using it. How about:

```suggestion
          Announcing GCN Classic Migration Survey and Retirement of Legacy Circular Email
          Submission Address. See{' '}
```